data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,88 @@ mission "FW Epilogue: Danforth"
+			`	As his tangent about his new invention continues, you think to yourself about the old times and the war, and all the people you've met along the way.`
+			`	"How about we get out of here <first>? My work here is done today, I know a homely little bar not far from here."`
+			choice
+				`	"Of course, it's not like there's a war to worry about anymore."`
+			`"Ha well of course, the war kept all of us plenty busy." He leads you out of the facility, all the while telling an anecdote that must be humor for an engineer.`
+			`	You follow Barmy into this bar he was telling you of, even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks, Barmys order catches you off guard and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle and you both tell stories of your early lives, specifically before leaving the ground. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
+			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there, I was lucky to get out myself"`
+			`	"No way you're from New Boston too! What are the chances?" you hastily respond.`
+			`	He chuckles, "Small galaxy I suppose." You both have a great time hearing of each others early lives but you wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and you head back to your ship.`
+				decline
+		event "plasma repeater advanced" 95
+		event "plasma repeater basics" 155
+	npc disable save
+		government "Test Dummy"
+		personality disables heroic staying
+		ship "Leviathan (Plasma Repeater)" `Hunk`
+	on complete
+		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon, increased fire rate even at lower damage per hit comes out with better overall results.`
+		conversation
+			scene "scene/plasma_scene"
+			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with a large gun much like an old Plasma Cannon, but somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs alot hotter, you see," he says. "But the results are impressive."`
+			choice
+				`	"What was that?"`
+					goto what
+				`	"It looks impressive, but it didn't do too much damage."`
+					goto damage
+			label what
+			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That, boy, was the Plasma Repeater. Of course, it wasn't easy to find where opportunity lies when everyone could have taken a crack at it."`
```

Comment:
```suggestion
			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That was the Plasma Repeater. Of course, it wasn't easy to find where opportunity lies when everyone could have taken a crack at it."`
```
`boy` specifies gender. Big no-no. Also, what does the last sentence mean? Don't quite understand it.

Reply:
The last sentence was a bit shaky, I admit. Rewording it to make sense. How does `Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."` sound?

Reply:
`countless fools` sounds a bit harsh, but it could also just be Barmy's character. Regardless, sounds good!